### DISCUSSION TITLE
Is WalletConnect V2 ready for production use?

### DISCUSSION THREAD

**Discussion by predragnikolic** [WalletConnect/walletconnect-monorepo#1239] [Developer Support] [answered]
Would you recommend WalletConnect V2 for production use for dapps because it is still in beta?

We are starting a project,
and discussing if it is worth jumping right in to V2 (the main benefit that we see is it is chain agnostic).

**Comment by chadyj** [comment, marked as answer]
For wallets, we suggest integrating both v1 and v2. There are several wallets already in production.

For dapps, some communities have already started using v2, such as Stellar.  It really depends on the wallet support.
